Source/ACE.Database/CharacterDatabase.cs

```diff
@@ -382,6 +397,24 @@ public void SaveCharacterProperties(DbObject dbObject, DatabaseTransaction trans
                 transaction.AddPreparedStatement(CharacterPreparedStatement.CharacterPropertiesStringInsert, dbObject.Id, (ushort)prop.Key, prop.Value);
         }
 
+        public void UpdateCharacterProperties(DbObject dbObject, DatabaseTransaction transaction)
```

Comment:
Sends ALL of the character properties to the database. This function utilizes the SQL "update" syntax.